### ISSUE TITLE
As an Industry reporter I want to see who edited the report last in the operations list so that I am aware

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#235]
#### Description:

This involves creating the 'Last edited by' column from the reporting dashboard wireframes. It captures who last hit 'save' or submit. May need to check the status of the save-as-you-go feature's implementation.

[Wireframe here](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7789%3A2031&t=JjzBvwOdyb7G91xp-1)

Blocked by: https://github.com/bcgov/cas-reporting/issues/243

#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by patriciarussellCAS** [comment]
We think this is already set up in the backend, so just need to build the UI/front-end. How do we expose this data in the frontend? 
Tagging @pbastia for refinement next week. 
This ticket is affected by how we're considering Registration data in our snapshot.

**Comment by suhafa** [comment]
Question for BA, do edits on registration while I am reporting relfect back as changes updated here?

**Comment by patriciarussellCAS** [comment]
Merged this ticket into #234 so will close this now.
